tools/src/test/scala/caliban/tools/SchemaWriterSpec.scala

```diff
@@ -34,31 +35,37 @@ object SchemaWriterSpec extends DefaultRunnableSpec {
               }
             |""".stripMargin
 
-        val typeCaseClass = Parser
-          .parseQuery(schema)
-          .map(_.objectTypeDefinitions.map(SchemaWriter.writeObject).mkString("\n"))
-          .flatMap(Formatter.format(_, None).map(_.trim))
-
-        val typeCaseClassArgs = Parser
-          .parseQuery(schema)
-          .map { doc =>
-            (for {
-              typeDef      <- doc.objectTypeDefinitions
-              typeDefField <- typeDef.fields
-              argClass      = SchemaWriter.writeArguments(typeDefField, typeDef) if argClass.nonEmpty
-            } yield argClass).mkString("\n")
-          }
-          .flatMap(Formatter.format(_, None).map(_.trim))
-
-        assertM(typeCaseClass)(
+        val typeCaseClass: ZIO[Blocking, Throwable, String] =
+          Parser
+            .parseQuery(schema)
+            .map(_.objectTypeDefinitions.map(SchemaWriter.writeObject).mkString("\n"))
+            .flatMap(Formatter.format(_, None).map(_.trim))
+
+        val typeCaseClassArgs: ZIO[Blocking, Throwable, String] =
+          Parser
+            .parseQuery(schema)
+            .map { doc =>
+              (for {
+                typeDef      <- doc.objectTypeDefinitions
+                typeDefField <- typeDef.fields
+                argClass      = SchemaWriter.writeArguments(typeDefField, typeDef) if argClass.nonEmpty
+              } yield argClass).mkString("\n")
+            }
+            .flatMap(Formatter.format(_, None).map(_.trim))
+
+        val a = assertM(typeCaseClass)(
           equalTo(
-            "case class Hero(name: HeroNameArgs () => String, nick: String, bday: Option[Int])"
```

Comment:
This test was apparently incorrect but was, for some reason, passing. 🤔

Not sure about the `andThen` following in the code